pnpm-lock.yaml

```diff
@@ -8607,7 +8607,7 @@ packages:
       debug: 4.3.4
       enhanced-resolve: 5.12.0
       eslint: 8.33.0
-      eslint-plugin-import: 2.27.5_fhrhggiruboehhi74bczdxbcti
+      eslint-plugin-import: 2.27.5_nowqz4jutkd4a233czbfk7jsgu
```

Comment:
Do you know why this changed by chance? Since dependencies weren't adjusted in a `package.json` I wouldn't have expected any `ppm-lock.yaml` file changes

Reply:
Maybe it's because I did `pnpm i` without `--frozen-lockfile`??

https://pnpm.io/cli/install#--frozen-lockfile

Reply:
I thought that was only needed in CI though

Reply:
any ideas @ynotdraw ? I wonder (since I removed the dependency) if I should just revert this change. Let me try that...